diff --git a/node_modules/gh-pages/lib/git.js b/node_modules/gh-pages/lib/git.js
index aa2b3dd..231bb35 100644
--- a/node_modules/gh-pages/lib/git.js
+++ b/node_modules/gh-pages/lib/git.js
@@ -27,7 +27,7 @@ util.inherits(ProcessError, Error);
  */
 function spawn(exe, args, cwd) {
   return new Promise((resolve, reject) => {
-    const child = cp.spawn(exe, args, {cwd: cwd || process.cwd()});
+    const child = cp.spawn(exe, args, {cwd: cwd || process.cwd(), stdio: 'inherit'});
     const buffer = [];
     child.stderr.on('data', chunk => {
       buffer.push(chunk.toString());
